editor/window_wrapper.cpp

```diff
@@ -454,14 +453,10 @@ ScreenSelect::ScreenSelect() {
 	// Create the popup.
 	const Size2 borders = Size2(4, 4) * EDSCALE;
 
-	popup = memnew(Popup);
+	popup = memnew(PopupPanel);
 	popup->connect("popup_hide", callable_mp(static_cast<BaseButton *>(this), &ScreenSelect::set_pressed).bind(false));
 	add_child(popup);
 
-	popup_background = memnew(Panel);
```

Comment:
You need to remove the declaration from `.h` file.

Reply:
@Giganzo This comment is the only thing keeping this PR from being merged. Will you have a chance to make the requested change sometime soon?

Reply:
Fixed.